.github/workflows/sonar.yaml

```diff
@@ -0,0 +1,54 @@
+name: sonar-analysis
+
+on:
+  push:
+    branches:
+      - master
+      - dev
```

Comment:
and tags

Reply:
@JosuaKrause  in jira ticket, definition of done only branches and prs are include. no tags. I will remove dev then.

Reply:
@mahadeveaswar  @TheAnkurGoswami  which branches other than master do you want to include in sonar analysis?

Reply:
Just master branch is fine here and include tags